### ISSUE TITLE
Docker-compose build not working - Invenio 3.0

### ISSUE THREAD

**Issue by mbachtell** [inveniosoftware/invenio#3856]
I've installed an instance via git but am unable to execute the docker-compose build command.  The error returned is "Can't find a suitable configuration file in this directory or any parent. Are you in the right directory? Supported filenames: docker-compose.yml, docker-compose.yaml"

I'm following the Docker instructions on http://invenio-software.org/gettingstarted.

**Comment by danschmidt5189** [comment]
Indeed, there are no Dockerfiles in the repo or compose configurations. What gives?

**Comment by danschmidt5189** [comment]
Actually there's not even any python code in this project. Is this a fake repo?

**Comment by lnielsen** [comment]
Project is fully alive, and the problems you are encountering is because we're preparing for final v3.0 release. Our next development sprint is May 14-25, and we expect the final release latest May 30th.

Invenio is a bundle package that essentially points to of other subpackages that are maintained individually. See [setup.py](https://github.com/inveniosoftware/invenio/blob/master/setup.py#L32-L59).

We've removed docker intentionally from this package. It now exists in [cookiecutter-invenio-instance](https://github.com/inveniosoftware/cookiecutter-invenio-instance/). The module is under construction and will be finalized together with the final v3.0 release.

You can take it for a test drive like this:

```console
$ pip install cookiecutter
$ cookiecutter https://github.com/inveniosoftware/cookiecutter-invenio-instance.git
$ cd my-site
$ ./scripts/bootstrap
$ ./scripts/setup
$ ./scripts/server
```

It requires docker compose, npm and Python 2.7 or 3.5 (3.6 won't work yet due to a problem with NPM). Also, you might hit problems until we release the final version.

Note that there's no data model, so with above you just get a barebones Invenio installation that can't do much. You'll have to use something like https://github.com/inveniosoftware/cookiecutter-invenio-datamodel to setup a new datamodel. This package is also in development and will be released together with the final v3.0

**Comment by lnielsen** [comment]
https://inveniosoftware.org/#getstarted
